enterprise/api/v1/views.py

```diff
@@ -88,27 +89,23 @@ class EnterpriseCustomerViewSet(EnterpriseReadOnlyModelViewSet):
     queryset = models.EnterpriseCustomer.active_customers.all()
     serializer_class = serializers.EnterpriseCustomerSerializer
 
+    USER_ID_FILTER = 'enterprise_customer_users__user_id'
     FIELDS = (
         'uuid', 'name', 'catalog', 'active', 'site', 'enable_data_sharing_consent',
         'enforce_data_sharing_consent',
     )
     filter_fields = FIELDS
     ordering_fields = FIELDS
 
-    @detail_route(
-        permission_classes=[permissions.IsAuthenticated, IsServiceUserOrReadOnly, IsStaffUserOrLinkedToEnterprise],
-        authentication_classes=[JwtAuthentication, BearerAuthentication, SessionAuthentication],
-        throttle_classes=[ServiceUserThrottle],
```

Comment:
As far as I can tell, these were not necessary in the first place (although it could've been that these *were* necessary at first but code moved around and someone didn't realize to erase this); they already come just fine through our view set inheritance hierarchy (from `EnterpriseViewSet`).

Reply:
Sounds good, thanks!